fs-index/src/lib.rs

```diff
@@ -3,39 +3,29 @@
 extern crate lazy_static;
 extern crate canonical_path;
 
-pub mod errors;
-pub use errors::{ArklibError, Result};
+use fs_utils::errors::{ArklibError, Result};
 
-pub mod app_id;
-pub mod id;
 pub mod index;
-
 pub mod link;
 pub mod pdf;
 
-mod atomic;
 mod storage;
```

Comment:
Storages should be defined in their own crate, too: `fs-storage`

FS because they could be implemented with some other backend later.